packages/fxa-auth-server/lib/payments/paypal-processor.ts

```diff
@@ -222,7 +223,8 @@ export class PaypalProcessor {
    */
   private async makePaymentAttempt(invoice: Stripe.Invoice) {
     const customer = invoice.customer as Stripe.Customer;
-    if (invoice.amount_due === 0) {
+    const minAmount = STRIPE_MINIMUM_CHARGE_AMOUNTS[invoice.currency] || 50;
```

Comment:
This `50` seems arbitrary.  If we cannot find a min amount, then process the invoice.  If we get an error, then we'll know what needs to be fixed.

Reply:
Oh yes that makes more sense.